lib/site-builder.js

```diff
@@ -213,18 +213,36 @@ SiteBuilder.prototype.readOrWriteConfig = function(configExists) {
 
 SiteBuilder.prototype.jekyllBuild = function() {
   var jekyll = config.jekyll;
-  var args = ['build', '--trace', '--destination', this.buildDestination,
-    '--config', '_config.yml,_config_18f_pages.yml'];
+  var args = [];
 
   if (this.usesBundler) {
     jekyll = config.bundler;
-    args = ['exec', 'jekyll'].concat(args);
+    args = ['exec', 'jekyll'];
```

Comment:
Since the pattern here is:

1. define array `args`
2. redefine array `args` as orig + concat

I think it's safer to keep the `concat` here, rather than completely redefining `args`. (In case, for instance, you added some more `args` manipulation between  lines 216 and 220.